packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textEditorParts/mask.tests.js

```diff
@@ -2334,6 +2335,42 @@ QUnit.module('validation', {}, () => {
         textEditor.option('value', 'f');
         assert.notOk(textEditor.option('isValid'), 'mask with an invalid value should be invalid');
     });
+
+    QUnit.test('The validation error should be null after removing the mask (T1214604)', function(assert) {
```

Comment:
Actually other errors should still be presented
```suggestion
    QUnit.test('The invalid mask error should be restores after removing the mask (T1214604)', function(assert) {
```

Reply:
[9ca99a2ac52d2f3d8a9e4a907803422b706598ed]